README.md

```diff
@@ -172,6 +172,9 @@ var j = schedule.scheduleJob(rule, function(){
 - `month (0-11)`
 - `year`
 - `dayOfWeek (0-6) Starting with Sunday`
+- `tz`
```

Comment:
Specify "TZ database name", or an example would be more explanatory ;)

Reply:
Added an example, hope that's satisfactory! 😄